### ISSUE TITLE
macOS - Launch alacritty from alfred and trigger create tmux window before running command

### ISSUE THREAD

**Issue by windowswithoutborders** [alacritty/alacritty#4729]
So I'm using this alacritty specific apple script with alfred from here https://github.com/alacritty/alacritty/issues/2638#issuecomment-678112947 and because I have alacritty launch with tmux, when a new instance of alacritty is spawned it connects to the prexisting tmux session for the first window, so both instances are just duplicates of one another and the apple script doesn't anticipate that behavior so it doesn't work for my usecase.

I don't write applescript but I looked up [this resource](https://eastmanreference.com/complete-list-of-applescript-key-codes) for keycodes and tried edit the applescript to tell system events send key codes for space and control down (my prefix key combo), and then after that send the key code for 'c' to create a new tmux tab, but I couldn't figure it out.

Basically, adapt apple script to:
- if alacritty already open
- send key codes for control and space (my prefix key combo)
- send key code for c (create new tmux tab)
- run command sent from alfred

Would really appreciate any help with this. Thanks

**Comment by chrisduerr** [comment]
I don't think Alacritty is the correct place to search for help with AppleScript. I have never used it and it has nothing to do with Alacritty. Id recommend searching for a place that actually knows about the tool you're seeking help with.
